test/benchmark/main.h

```diff
@@ -0,0 +1,7 @@
+#pragma once
+// NOLINT(namespace-envoy)
+
+/**
+ * Benchmarks can use this to skip or hurry through long-running tests in CI.
+ */
+bool skipExpensiveBenchmarks();
```

Comment:
Nit: Can you put this in some namespace, e.g. `envoy::benchmark`? Otherwise this is fine, the blast radius of the static is limited to just benchmarks.

Reply:
Done, assuming you weren't intentionally case sensitive.